src/packer.pkr.hcl

```
@@ -124,19 +124,20 @@ build {
   sources = ["source.amazon-ebs.freeipa"]
 
   provisioner "ansible" {
-    extra_arguments = ["--extra-vars", "ansible_python_interpreter=auto"]
-    playbook_file   = "src/upgrade.yml"
+    playbook_file = "src/upgrade.yml"
+    use_sftp      = true
   }
 
   provisioner "ansible" {
-    extra_arguments = ["--extra-vars", "ansible_python_interpreter=auto"]
-    playbook_file   = "src/python.yml"
+    playbook_file = "src/python.yml"
+    use_sftp      = true
   }
 
   provisioner "ansible" {
     ansible_env_vars = ["AWS_DEFAULT_REGION=${var.build_region}"]
     extra_arguments  = ["--extra-vars", "{ansible_python_interpreter: auto, build_bucket: ${var.build_bucket}}"]
```

Based on your [earlier comment](https://github.com/cisagov/freeipa-server-packer/pull/97#discussion_r1024279288) should this be changed like so?

```suggestion
    extra_arguments  = ["--extra-vars", "build_bucket=${var.build_bucket}"]
```

---

Yes, you are correct.